### ISSUE TITLE
Add function arguments to database and API

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#2533]
Currently function arguments are managed by plugins and they add argument blocks to disassembly during analysis. It only works in disassembly view, not in Graph, Arguments and Call Stack view, and it is impossible to share the information with another plugin.
By adding function arguments to database and API we could support these usages. We can start with only argument names. The easiest implementation would be a new "argument" string associated with the function address. The string could be a JSON or comma separated strings describing all arguments. The information would be provided by a plugin or the user.

**Comment by mrexodia** [comment]
I’m not really interested in doing this improperly. Inside the x64dbg type
system there is support for function arguments (with types) but there is no
type database available to make use of it. A first step would be to parse
all Windows headers/documentation and go from there.

**Comment by torusrxxx** [comment]
So is there a way to link a type to a label? Then we can display all arguments in call stack.

**Comment by mrexodia** [comment]
Yeah I’m pretty sure there is a way to go from name to function type, so
that way the label could be used to get the type...

**Comment by torusrxxx** [comment]
It seems we need two types of "type", one provided by the Windows SDK and one defined by the user, just like we need symbols as well as labels.
